libs/features/components/banners/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="container" [ngStyle]="{'background-color': bgcolor}">
```

Comment:
This component has already been implemented. Sync the latest changes and work with what is already there.

Reply:
Yes the component has already been implemented maybe i just added a space by mistake but i have not modified it